frontend/src/app/modules/settings/team/team-members/team-members.component.ts

```diff
@@ -55,7 +59,23 @@ export class TeamMembersComponent extends BaseComponent implements OnInit, OnDes
             this.teamId = params['id'];
         });
 
+        this.isCurrentUserHasMemberRole();
+    }
+
+    isCurrentUserHasMemberRole() {
         this.getTeamMembers();
+
+        this.userService
+            .getCurrentUser()
```

Comment:
Do you need to call api here? 
You need to get only current user id, I think you can use  this.userService .userChangedEvent$.